cmd/run.go

```diff
@@ -332,11 +365,28 @@ func runServer(ctx context.Context, config *Config) error {
 
 	logger := buildLogger(config.Log.Format)
 	tracer := telemetry.NewNoopTracer()
-	meter := telemetry.NewNoopMeter()
 	tokenEncoder := encoder.NewBase64Encoder()
 
-	var datastore storage.OpenFGADatastore
 	var err error
+	var meter metric.Meter
+	switch config.Metrics.MetricsExporter {
+	case "prometheus":
+		meter, err = telemetry.NewPrometheusMeter()
+		if err != nil {
+			return fmt.Errorf("failed to generate prometheus exporter: %w", err)
+		}
+	case "otlp":
+		meter, err = telemetry.NewOTLPMeter(logger, ctx, config.Metrics.OTLP.Protocol, config.Metrics.OTLP.Endpoint)
+		if err != nil {
+			return fmt.Errorf("failed to generate otlp exporter: %w", err)
```

Comment:
Very minor, but i'm confused by the naming - the variable is called `meter` but the error says `exporter`

Reply:
Indeed, I updated the error message.